cmd/brew-bootstrap-asdf

```diff
@@ -24,28 +24,35 @@ cleanup() {
 }
 trap "cleanup" EXIT
 
+log_step() {
+  echo "==> $1"
```

Comment:
I don't think we need this enabled all the time. Can you set it to log only in debug mode?

```suggestion
  if [ "x$PRINT_DEBUG" = "x1" ]; then
    echo "==> $1"
  fi
```